api/types/user.ts

```diff
@@ -1,9 +1,9 @@
 type User = {
```

Comment:
Seems to be this file can be removed entirely.

Reply:
done